examples/fabric-admin/commands/pairing/PairingCommand.cpp

```diff
@@ -544,6 +546,8 @@ void PairingCommand::OnCurrentFabricRemove(void * context, NodeId nodeId, CHIP_E
         fprintf(stderr, "Device with Node ID: 0x%lx has been successfully removed.\n", nodeId);
 
 #if defined(PW_RPC_ENABLED)
+        chip::app::InteractionModelEngine::GetInstance()->ShutdownSubscriptions(command->CurrentCommissioner().GetFabricIndex(),
+                                                                                nodeId);
```

Comment:
Do we also need to remove the subscription from mDeviceSubscriptionMap here? or this can be done automatically?

Reply:
It is done automatically via the callbacks from `ReadClient`'s `OnDone` which call `DeviceSubscriptionManager::DeviceSubscriptionTerminated`

I have confirmed this via logging